routes/web.php

```diff
@@ -8,11 +8,14 @@
 
 Route::get('posts/create', 'HomeController@createForm');
 Route::post('posts/create', ['uses' => 'GistsController@storeAndRedirect', 'as' => 'post.create']);
+Route::get('posts/{gistId}/star', ['middleware' => ['auth'], 'uses' => 'GistsController@star', 'as' => 'post.star']);
```

Comment:
One thing you could do here is POST to this URL to star it and DELETE to this URL to unstar it

Reply:
For now, I'm leaving this as-is since it will require some additional logic in the view to process the request method. Let me know if you feel strongly about making this change though, @mattstauffer. There may be something I'm not seeing.

```
toggleStar() {
      // we'd have to adjust this method to generate a post/delete request based on the value of isStarred
      axios
        .get(this.toggleRoute())
        .then((response) =>
          this.isStarred ? (this.isStarred = false) : (this.isStarred = true)
        );
    },
```

Reply:
I haven't done it in a while, but I *think* you'd do this:

```
axios({
  method: this.isStarred ? 'delete' : 'post'
  url: this.toggleRoute(),
}).then
```

Reply:
This worked perfectly! Thanks for the tip.